Tools/TestWebKitAPI/Tests/WebKitCocoa/FullscreenVideoTextRecognition.mm

```diff
@@ -302,6 +314,44 @@ - (void)waitForImageAnalysisToEnd
     Util::run(&doneWaiting);
 }
 
+// FIXME: Re-enable this test for iOS once webkit.org/b/248094 is resolved
```

Comment:
Does that bug apply for this test too?

Reply:
ya, it applies for all tests in this file